cms/topics/models.py

```diff
@@ -223,7 +232,7 @@ def processed_methodologies(self) -> list[MethodologyPage]:
         return highlighted_pages + pages
 
     @cached_property
-    def table_of_contents(self) -> list[dict[str, str | object]]:
+    def table_of_contents(self) -> list[dict[str, Any]]:
```

Comment:
we do need to update the table of contents to include a "Data" item

Reply:
Done in [d250c72](https://github.com/ONSdigital/dis-wagtail/pull/226/commits/d250c729fc2e0788b15e0d4efeb021e16bb166d8)